recipes/z5py/bld.bat

```diff
@@ -0,0 +1,18 @@
+mkdir build
+cd build
+
+set CONFIGURATION=Release
+
+cmake .. -G "%CMAKE_GENERATOR%" -DCMAKE_PREFIX_PATH=%PREFIX% ^
+    -DCMAKE_INSTALL_PREFIX="%LIBRARY_PREFIX" ^
+    -DBOOST_ROOT=%LIBRARY_PREFIX% ^
```

Comment:
This may be totally unrelated, but I have had problems building things on Windows in the past when the variable type is not specified. Normally this is worse for paths, but it does occur other places as well. Doing something like [this]( https://github.com/conda-forge/gflags-feedstock/blob/master/recipe/bld.bat#L10 ) often fixes the issue.